### DISCUSSION TITLE
Where to find the main feed of SBS TV?

### DISCUSSION THREAD

**Discussion by AVGCFan1995** [iptv-org/community#983] [Q&A]
@freearhey @alvarosamudio @BellezaEmporium @trivjedn00m @Alstruit @sunyuanzhe

Hey, guys! Did you find the main feed of SBS TV since I couldn't find it online but only the news and regional feeds do exist?

#EXTINF:-1 tvg-id="HLDRDTV.kr",SBS CJB (540p) [Not 24/7]
http://1.222.207.80:1935/live/cjbtv/playlist.m3u8
#EXTINF:-1 tvg-id="HLCGDTV.kr",SBS G1 (360p) [Not 24/7]
http://61.82.49.4:1935/live/_definst_/tv.stream/playlist.m3u8
#EXTINF:-1 tvg-id="HLKJDTV.kr",SBS JIBS (720p) [Not 24/7]
http://123.140.197.22/stream/1/play.m3u8
#EXTINF:-1 tvg-id="HLDQDTV.kr",SBS JTV (406p) [Not 24/7]
http://61.85.197.53:1935/jtv_live/myStream/playlist.m3u8
#EXTINF:-1 tvg-id="HLDHDTV.kr",SBS KBC (1080p) [Not 24/7]
http://119.200.131.11:1935/KBCTV/tv/playlist.m3u8
#EXTINF:-1 tvg-id="HLDGDTV.kr",SBS KNN (1080p) [Not 24/7]
https://stream1.knn.co.kr/hls/9ly4534y7dm2xfa123r2_tv/index.m3u8
#EXTINF:-1 tvg-id="HLDGDTV.kr",SBS KNN (450p) [Not 24/7]
http://211.220.195.200:1935/live/mp4:KnnTV.sdp/playlist.m3u8
#EXTINF:-1 tvg-id="SBSNews.kr",SBS News (1080p) [Not 24/7]
https://ythlsgo.onrender.com/channel/UCkinYTS9IHqOEwR1Sze2JTw.m3u8
#EXTINF:-1 tvg-id="HLDEDTV.kr",SBS TBC (540p) [Geo-blocked] [Not 24/7]
http://203.251.91.122:1935/on-air-Backup/tv/playlist.m3u8
#EXTINF:-1 tvg-id="HLDEDTV.kr",SBS TBC (540p) [Geo-blocked] [Not 24/7]
https://tistory3.daumcdn.net/tistory/2864460/skin/images/Public_8.m3u8
#EXTINF:-1 tvg-id="HLDFDTV.kr",SBS TJB (1080p) [Not 24/7]
http://1.245.74.5:1935/live/tv/.m3u8
#EXTINF:-1 tvg-id="HLDPDTV.kr",SBS UBC (540p) [Not 24/7]
http://59.23.231.102:1935/live/UBCstream/playlist.m3u8

Here's the links for SBS TV are either dead or alive when it's either working or not working on VLC media player please.

SBS,http://221.157.125.239:1935/live/psike/playlist.m3u8
SBS,http://183.99.22.189:9981/stream/channelid/1124346476
SBS,http://220.95.92.160:9981/stream/channelid/1445999617
SBS,https://apichvod19.seezntv.com/OTM/10550_M.m3u8?AdaptiveType=HLS&SessionID=3438d7e9-3e70-43b1-9fe6-b8290cd25403
SBS,http://1.245.74.5:1935/live/tv/.m3u8
SBS,http://itskoi.dothome.co.kr/sbs.php?id=S01
SBS,https://61ff3340258d2.streamlock.net/jtv_live/myStream/playlist.m3u8
SBS,http://ye23.win/iptv/sbs5.1.php?id=S01
SBS,http://han.ddkdxmkj.com/api/sbs.php?id=1
SBS,http://222.109.117.27:9981/stream/channelid/2096368258

Can you find it please?
![Seoul_Broadcasting_System_logo svg](https://github.com/iptv-org/iptv-org.github.io/assets/143818591/4ba869ab-dc0b-4405-b64f-a4d9fe2d47ae)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
